sourcespec/ssp_build_spectra.py

```diff
@@ -408,6 +408,45 @@ def _build_weight_from_frequency(config, spec):
     return weight
 
 
+def _build_weight_from_inv_frequency(config, spec, pow=0.25):
+    """
+    Build spectral weights from inverse frequency (raised to a power < 1)
+    """
```

Comment:
Should we raise `ValueError` if `pow>=1` ?

Reply:
We could do that. `pow` could be a user-configurable option, but it's not currently implemented that way.

Reply:
Yes, but it's better to make a check, in case we want to make it user-configurable in the future.